Project/src/MakeCall/AddParticipantPopover.js

```diff
@@ -43,8 +48,9 @@ export default function AddParticipantPopover(props) {
                             <h3 className="add-participant-panel-header">Add a participant</h3>
                             <div className="add-participant-panel-header">
                                 <TextField className="text-left" label="Identifier" onChange={e => setUserId(e.target.value)} />
+                                <TextField className="text-left" label="Thread Id (Needed if SDK is initialized for Teams User )" onChange={e => setThreadId(e.target.value)} />
                                 <TextField className="text-left" label="Alternate Caller Id (For adding phone number only)" onChange={e => setAlternateCallerId(e.target.value)} />
-                                <Button className="mt-3" onClick={handleAddCommunicationUser}>Add CommunicationUser</Button>
+                                <Button className="mt-3" onClick={handleAddCommunicationUser}>Add Participant</Button>
```

Comment:
nit. Do we want to also update handleAddCommunicationUser to be handleAddParticipant? do we still want that console log that says handleAddCommunicationUser?